OneCommunityGlobal/HighestGoodNetworkApp
# Aaron hotfix white screen issue for viewing user profiles

# Description
Fixing issue where going to some user profiles cause a white screen

## Related PRS (if any):
N/A

## Main changes explained:
- added check for users who don't have media links

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin/owner user
4. go to dashboard→ Other Links → User Management
5. verify you are are able to view the profiles of users with the following media links:
  - have both Google Doc and Media Folder
  - have Google Doc but not Media Folder
  - have Media Folder but not Google Doc
  - have no media links 

## Screenshots or videos of changes:

<img width="960" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/10265513/e4ea0611-edcd-47ef-85e5-da09b3e2e065">
